source/includes/_node.md

```diff
@@ -2,106 +2,216 @@
 
 ## JSON-RPC Requests 
 
-```shell
-
-```
-
-```javascript
-
-```
+Route for JSON-RPC requests, most of which mimic the bitcoind RPC calls completely.
 
-> The above command returns JSON structured like this:
+### HTTP Request
+`POST /`
 
-```json
-[
-  
-]
-```
-Route for JSON-RPC requests, most of which mimic the bitcoind RPC calls completely.
 
 
 ## Get server info
 
-```shell
+```shell--curl
+curl $url/
+```
 
+```shell--cli
+bcoin cli info
 ```
 
 ```javascript
-
+const info = await client.getInfo();
 ```
 
 > The above command returns JSON structured like this:
 
 ```json
 {
-  "version": "1.0.0-alpha",
-  "network": "regtest",
+  "version": "v1.0.0-beta.14",
+  "network": "testnet",
   "chain": {
-    "height": 177,
-    "tip": "18037e4e4906a6c9580356612a42bb4127888354825f4232d811ef7e89af376a",
-    "progress": 0.9935725241578355
+    "height": 1157058,
+    "tip": "00000000000002ac70408966be53a1e01e7e014a3d4f1f275201c751de7d6e77",
+    "progress": 1
   },
   "pool": {
-    "host": "96.82.67.198",
-    "port": 48444,
-    "agent": "/bcoin:1.0.0-alpha/",
+    "host": "203.0.113.114",
+    "port": 18333,
+    "agent": "/bcoin:v1.0.0-beta.14/",
     "services": "1001",
-    "outbound": 1,
-    "inbound": 1
+    "outbound": 8,
+    "inbound": 0
   },
   "mempool": {
-    "tx": 1,
-    "size": 2776
+    "tx": 39,
+    "size": 121512
   },
   "time": {
-    "uptime": 6,
-    "system": 1486696228,
-    "adjusted": 1486696228,
-    "offset": 0
+    "uptime": 7403,
+    "system": 1502381034,
+    "adjusted": 1502381035,
+    "offset": 1
   },
   "memory": {
-    "rss": 63,
-    "jsHeap": 24,
-    "jsHeapTotal": 34,
-    "nativeHeap": 28
+    "total": 87,
+    "jsHeap": 23,
+    "jsHeapTotal": 30,
+    "nativeHeap": 56,
+    "external": 8
   }
 }
 ```
+### HTTP Request
+`GET /`
 
 
-## Get mempool snapshot
 
+## Get mempool snapshot
 
-```shell
+```shell--curl
+curl $url/mempool
+```
 
+```shell--cli
+bcoin cli mempool
 ```
 
 ```javascript
+const mempoolTxs = await client.getMempool();
```

Comment:
I was thinking about it, decided not include. I thought of describing the basic usage (how to create Client and how to wrap it on separate doc). 

Not sure if we need 10 lines of code(or even more if you want to initialize bcoin.Config)